sqlx-core/src/postgres/connection/mod.rs

```diff
@@ -96,6 +98,19 @@ impl Debug for PgConnection {
     }
 }
 
+impl CachingConnection for PgConnection {
+    fn cached_statements_count(&self) -> usize {
+        self.cache_statement.len()
+    }
+
+    fn clear_cached_statements(&mut self) -> BoxFuture<'_, Result<(), Error>> {
+        Box::pin(async move {
+            self.cache_statement.clear();
```

Comment:
Couldn't find anything about deletion of statements in PostgreSQL. Is it needed?